price-feeder/oracle/util_test.go

```diff
@@ -77,3 +78,31 @@ func TestComputeVWAP(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
We need to implement a test for having multiple providers and where one fails. I think you can build off the existing tests. Lmk  if you need a hand on this